### PR TITLE
Debug LDAP

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

**Comment by arnaud-morvan** [comment]
@RemiDesgrange : here are my changes to debug LDAP

File: docker-compose.override.sample.yaml [hunk 1]
```diff
@@ -7,4 +7,9 @@ services:
       context: .
       target: tester
     volumes:
-      - ./extension:/src/extension
+      - ./runtime/docker-entrypoint.d:/docker-entrypoint.d
+      - /home/amorvan/dev/nextcloud/server/apps:/var/www/html/apps:ro
+      - /home/amorvan/dev/nextcloud/server/lib:/var/www/html/lib:ro
```
Comment (RemiDesgrange) [inline docker-compose.override.sample.yaml]: Maybe remove your name and put a relative path for the nexcloud server  place ?

File: runtime/docker-entrypoint.d/02-install-extension [hunk 2]
```diff
@@ -8,6 +10,8 @@ run_as() {
     fi
 }
 
+run_as "php occ config:system:set --type=\"integer\" --value=\"${LOG_LEVEL}\" \"loglevel\""
+
```
Comment (RemiDesgrange) [inline runtime/docker-entrypoint.d/02-install-extension]: This is not a desired config in production.

Comment (arnaud-morvan) [reply runtime/docker-entrypoint.d/02-install-extension]: Ok let's keep it only in readme so.

File: docker-compose.override.sample.yaml (hunk 1 above)
Comment (arnaud-morvan) [reply docker-compose.override.sample.yaml]: Done with some sort of standard relative path.

**Comment by arnaud-morvan** [comment]
@RemiDesgrange : Thanks for review, should be ready for merge now.

**Review by RemiDesgrange — APPROVED** [review]
:shipit:

**Comment by arnaud-morvan** [comment]
Note that I do not have rights to merge PR in this repo.

**Pull request merged**